pages/views/roles/data.md

```diff
@@ -0,0 +1,80 @@
+
+#### Long-term Copies and Backups
+
+Backups are commonly recommended for most data, and certainly all important data. Guidance like the 3-2-1 rule suggest keeping multiple copies, across multiple media and multiple sites. This will increase the carbon cost of storage accordingly, and should be accounted for.
+
+Removable media may also be considered for long-term storage. Their intermittent use makes it harder estimate energy use while active, though published data imply that energy use is significantly lower than that for fixed drives. Their embedded carbon may still be measured:
+
+| Storage Type       | Embedded Carbon (kgCO2e) | Media Embedded Carbon (kgCO2e/TB) |
+| ------------------ | -----------------------: | --------------------------------: |
+| Tape Drive (LTO 8) | 46.6                     | 0.62                              |
+
+### Indirect
+
+{% include categoryLabel.html label="CatC" %}
+
+{% include categoryItem.html item="CatCManaged" id="Managed" %}
+
+Much data is now stored on cloud storage - that is, storage managed by a cloud provider and accessed over the internet. Almost all cloud storage exists, ultimately, in HDD or SSD, and so while embedded carbon costs are hidden from the user, they are still incurred by the provider.
+
+Cloud storage provision also incurs a carbon cost from use, and this can be more visible to the end user. Some cloud providers allow users to examine estimated emissions from their services, but not all do; still, emissions can be estimated since they are roughly proportional to the amount of data stored. Storing one terabyte of data on a major cloud storage provider for one year uses around 35 kilowatt-hours of electricity.
+
+It should be noted that, by default, most cloud providers duplicate data, keeping three copies on three different machines (to ensure availability and insure against drive failure).
+
+#### Location and Legislation
+
+Since users can typically select different regions for their cloud service hosting, they can select regions with different carbon intensities and so different resultant emissions. However, there may be data localisation laws, which limit the sites at which personal data may be stored.
+
+## Downstream Emissions
+
+{% include categoryLabel.html label="CatD" %}
+
+{% include categoryItem.html item="CatDEndUserDevices" id="EndUserDevices" %}
+
+The same arguments for storage on servers (see above) apply to user devices (though embodied carbon can no longer be measured). However, electricity use for storage can be estimated, using the knowledge of the type and size of data stored, and the likely storage medium. (For instance, smartphones store data in flash storage, while laptops and desktops may use SSDs or HDDs.)
```

Comment:
I'm not sure I understand why "embodied carbon can no longer be measured"?

Reply:
Basically, since you don't have control over user's devices, and they're likely to be shared in a fashion that's very hard to estimate. That means that without rather intrusive monitoring, you'd have to guess end user device energy consumption, rather than measure it.

I guess I could try and rework the above into the text, but I wonder if that's something that should live outside of the data page if it's missing in the TCS.

Reply:
The statement (**embodied** carbon can no longer be measured), we can use reasonable averages to assume embodied carbon of devices (laptops, phones, external storage devices)  - unless you are specifically referring the the actual storage component, ie the actual physical hard drives?? In which case the user device should already encompass that anyway, ignoring edge cases where users replace/upgrade those drives.

Reply:
I've made a change in response to this. However, I wonder if there's a disconnect here: I'm imagining this overlay being used to estimate the carbon costs of data (only) in a project, which might not be an actual use case for it, and I suspect that you're looking at the TCS (no overlay) as being the actual approach.